data/mods/Magiclysm/effects/effects.json

```diff
@@ -460,5 +460,157 @@
     "name": [ "Lightness" ],
     "desc": [ "You feel yourself a bit lighter.  You doubt you can lose weight from this." ],
     "base_mods": { "speed_mod": [ 15 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "flask_speed_eff",
+    "name": [ "Velocitas" ],
+    "desc": [ "Make you faster, and allow to hold this state for a while." ],
+    "apply_message": "You feel your legs can work much better than before",
+    "remove_message": "Your speed slows down.",
+    "rating": "good",
+    "base_mods": { "speed_mod": [ 40 ], "stamina_min": [ 20 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "flask_healing_eff",
+    "name": [ "Sanatio" ],
+    "desc": [ "Improve your organism's ability to heal himself." ],
+    "//": "i have no idea how health works now, so ill just copypaste the nature commune effect and stretch it.",
+    "base_mods": { "health_min": [ 1 ], "health_chance": [ 2880 ], "h_mod_min": [ 1 ], "h_mod_chance": [ 5760 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "flask_stamina_eff",
+    "name": [ "Constantia" ],
+    "desc": [ "You can't get tired." ],
+    "apply_message": "You feel an energy in your muscles, untired and jazzed.",
+    "remove_message": "The energy in your body fades away.",
+    "base_mods": { "stamina_min": [ 50 ] }
+  },
+  {
+    "//": "TODO: transform all stuff below this comment to effects when there would be a way to use enchantments with effect mechanic",
```

Comment:
It seems like #60161 has allowed effects to have enchantments

Reply:
oh, i thought it works by EoC. need to rewrite it now